sparse/mlir_backend/tests/test_simple.py

```diff
@@ -115,3 +120,10 @@ def test_add(rng, dtype):
     expected = a + c
     assert isinstance(actual, np.ndarray)
     np.testing.assert_array_equal(actual, expected)
+
+    # TODO: Blocked by https://github.com/llvm/llvm-project/issues/107477
+    # d = sps.random_array(SHAPE, density=DENSITY, format="coo", dtype=dtype, random_state=rng)
+    # d_tensor = sparse.asarray(d)
+    # actual = sparse.add(b_tensor, d_tensor).to_scipy_sparse()
+    # expected = b + d
+    # np.testing.assert_array_equal(actual.todense(), expected.todense())
```

Comment:
Does this still not work?

Reply:
Yes, adding COO tensors and returning the result as COO is broken IMO and explained in detail here: https://github.com/llvm/llvm-project/issues/107477